orcid-core/src/main/java/org/orcid/core/manager/validator/ActivityValidator.java

```diff
@@ -155,22 +155,6 @@ public static void checkExternalIdentifiersForDuplicates(ExternalIDs newExtIds,
         }
     }
 
-    private static boolean isDupRelationship(ExternalID newId, ExternalID existingId) {
-        return existingId.getRelationship() != null && existingId.getRelationship().equals(Relationship.SELF) && newId.getRelationship() != null
-                && newId.getRelationship().equals(Relationship.SELF);
```

Comment:
@TomDemeranville this will break. Identifiers that are part of can exist across multiple activities. @wjrsimpson @amontenegro can you help Tom.